bar_app/serializers.py

```diff
@@ -0,0 +1,169 @@
+from rest_framework import serializers
+from rest_framework.exceptions import ValidationError
+from django.contrib.auth.models import User
+from .models import *
+
+class UserSerializer(serializers.ModelSerializer):
```

Comment:
this is not needed in the first place. Why is it here ?

Reply:
Initially I just wanted to get the user's token, after that I thought it wouldn't be a bad idea to create a view for managing user registrations and logins, so I created UserSerializer there for user's fields.